packages/client/hmi-client/src/services/project.ts

```diff
@@ -178,33 +178,41 @@ async function getPermissions(projectId: Project['id']): Promise<PermissionRelat
 	}
 }
 
-async function setPermissions(projectId: Project['id'], userId: string, relationship: string) {
+async function setPermissions(
+	projectId: Project['id'],
+	userId: string,
+	relationship: string
```

Comment:
Should this use `PermissionProject["PermissionProject"]` type?
```suggestion
	relationship: PermissionProject["PermissionProject"]
```

Reply:
Don't know - I didn't write this, but IntelliJ insisted this get reformatted for no good reason, all because I changed the return type.